suggestions/keyboards.json

```diff
@@ -1 +1,8 @@
-[]
\ No newline at end of file
+[
+  {
```

Comment:
I suggest adding Indic Keyboard too. FlorisBoard does not yet support Indian languages.